shoop/themes/classic_gray/templates/classic_gray/products_view.jinja

```diff
@@ -0,0 +1,39 @@
+{% extends "shoop/front/base.jinja" %}
+
+{% block title %}{% trans %}Products{% endtrans %}{% endblock %}
+{% block content_title %}{% trans %}Products{% endtrans %}{% endblock %}
+
+{% block breadcrumb %}
+    <ol class="breadcrumb">
+        <li><a href="/">{% trans %}Home{% endtrans %}</a></li>
+        <li class="active">{% trans %}Products{% endtrans %}</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-3 categories-nav">
+            {% include "shoop/front/includes/_category_menu.jinja" %}
+        </div>
+        <div class="col-md-9">
+            <div class="row product-list-actions">
+                <div class="toggle-view">
+                    <span class="control-label">{% trans %}Show Grid/List{% endtrans %}</span>
```

Comment:
Well it was originally "Shop Grid/List" so I am not sure do we need those extra spaces?